pkg/statistics/handle/types/interfaces.go

```diff
@@ -324,6 +324,13 @@ type PartitionStatisticLoadTask struct {
 // PersistFunc is used to persist JSONTable in the partition level.
 type PersistFunc func(ctx context.Context, jsonTable *statsutil.JSONTable, physicalID int64) error
 
+// MetaUpdate records a partition-level meta update.
```

Comment:
This comment might not be entirely accurate, as it also applies to non-partitioned tables.